src/results/results_table_inprocess.php

```
@@ -0,0 +1,195 @@
+    $asse   = $tds["asse"];
+    $words  = $tds["words"];
+    $refs   = $tds["refs"];
+    $qid    = $tds["qid"];
+    $title  = $tds["title"];
+    $_user_ = $tds["user"];
+    $_date_ = $tds["date"];
+    //---
+    // $cnt2 = $full ? "$cnt.Full" : $cnt;
+    $cnt2 = $full && (strtolower(substr($title, 0, 6)) != 'video:') ? "$cnt.Full" : $cnt;
+    //---
+    $tab_th = ($tra_btn == "1") ? "<th>$tab</th>" : "";
```

PHPMD (2.15.0)

Avoid unused local variables such as '$tab_th'. (undefined)

(UnusedLocalVariable)